src/GameState/index.tsx

```diff
@@ -121,6 +147,18 @@ export default function GameState({ navigation, route, player }: any) {
             });
             break;
 
+          case 'animatedMarble':
+            if (messageParsed.event.data.playerId != player.id) {
+              dispatch({
+                type: 'animatedMarble',
+                value: {
+                  marblesCoordinate: messageParsed.event.data.marblesCoordinate,
+                  direction: messageParsed.event.data.direction,
+                },
+              });
+            }
+            break;
+
```

Comment:
It seems there is to much logic in cases of a switch

Reply:
I agree. But for the sake of time I wouldn't change the switch for now.